server/advanced/byo-proxy-service-guides/tailscale-vpn-setup.md

```diff
@@ -1,20 +1,52 @@
+# Tailscale Funnel
+
+Tailscale is a mesh VPN software that uses WireGuard technology. It also include other fun features like Tailscale Funnel.
+
+[Tailscale Funnel ](https://tailscale.com/kb/1223/tailscale-funnel/) allows you to publicly expose your machine's local services without needing to purchase a domain & set up port forwarding. It hosts your machine's domain on their Funnel Servers. The Funnel Server accepts requests & sends a TCP proxy to your machine where TLS cert is terminated. Simple, secure & only requires a few short commands.
+
+--- 
+Requirements
+- [Tailscale account](https://login.tailscale.com/start)
+- Enable [HTTPS certificates](https://login.tailscale.com/admin/dns) in the [admin console](https://login.tailscale.com/admin/dns).
+- Open the [**Access controls**](https://login.tailscale.com/admin/acls) page in the admin console and click the **Add Funnel to policy** button
 ---
-description: >-
-  This guide will show you how you can use your existing Tailscale VPN account
-  to connect to your BlueBubbles Server (via a Funnel), without port-forwarding
----
+1. Download Tailscale from the [Mac App Store](https://apps.apple.com/ca/app/tailscale/id1475387142) or [directly from Tailscale](https://pkgs.tailscale.com/stable/#macos) 
+2. Login from the top right menu icon & enable start on login from prefrences
+3. Add alias for the Tailscale CLI to your shell configuration by entering  the command below into terminal.
+```bash
+echo 'alias tailscale="/Applications/Tailscale.app/Contents/MacOS/Tailscale"' | sudo tee -a ~/.zshrc
+```
+Alternatively, you can use `/Applications/Tailscale.app/Contents/MacOS/Tailscale <command>` 
+
+3. Proxy requests to BlueBubbles's local web server on port 1234.
+
+```bash
+tailscale serve https / http://127.0.0.1:1234
+```
```

Comment:
Can you clarify that this port # could be different depending on what the user has set up (but indicate that its usually 1234)